h/services/search_index/_queue.py

```diff
@@ -93,7 +99,9 @@ def sync(self, limit):
             return
 
         annotation_ids = {
-            URLSafeUUID.hex_to_url_safe(job.kwargs["annotation_id"]) for job in jobs
+            URLSafeUUID.hex_to_url_safe(job.kwargs["annotation_id"])
+            for job in jobs
+            if not job.kwargs.get("force", False)
```

Comment:
There's no point in reading the annotation from Postgres or Elastic if `force` is `True`, we aren't going to use it